docs: task class: improve_docs — add cloud-only prerequisite clarification (#22508)
netdata/netdata: docs/netdata-oss-limitations.md

### Increase Node Limits

The 5-node limit on multi-node dashboards applies to both **Anonymous** and **Community** users.

1. **Upgrade to Business** for unlimited nodes in multi-node views
2. **Select preferred nodes** in Space settings to choose which 5 nodes appear in multi-node views

:::note

> Preferred nodes is a Netdata Cloud feature that requires signing in. It is **not available** to anonymous users accessing the local Agent dashboard directly.

Reviewer: ```suggestion
Preferred nodes is a Netdata Cloud feature that requires signing in.
```

### Increase Node Limits

The 5-node limit on multi-node dashboards applies to both **Anonymous** and **Community** users. You can:

1. **Upgrade to a paid plan** for unlimited nodes in multi-node dashboards
2. **Select preferred nodes** in **Space Settings > Nodes** to choose which 5 nodes appear in multi-node dashboards

:::note

> Preferred node selection only affects Netdata Cloud dashboards. On the local agent dashboard (accessed directly at `http://<agent-ip>:19999`), the nodes shown in multi-node views are determined by the agent's streaming configuration and cannot be changed via preferred node settings.

Reviewer: ```suggestion
Preferred node selection only affects Netdata Cloud dashboards. On the local agent dashboard (accessed directly at `http://<agent-ip>:19999`), the nodes shown in multi-node views are determined by the Agent's streaming configuration and cannot be changed via preferred node settings.
```

> Preferred node selection only affects Netdata Cloud dashboards. On the local agent dashboard (accessed directly at `http://<agent-ip>:19999`), the nodes shown in multi-node views are determined by the Agent's streaming configuration and cannot be changed via preferred node settings.

Reviewer: ```suggestion
Preferred node selection only affects Netdata Cloud dashboards. On the local Agent dashboard (accessed directly at `http://<agent-ip>:19999`), the nodes shown in multi-node views are determined by the Agent's streaming configuration and cannot be changed via preferred node settings.
```

> Preferred node selection only affects Netdata Cloud dashboards. On the local agent dashboard (accessed directly at `http://<agent-ip>:19999`), the nodes shown in multi-node views are determined by the agent's streaming configuration and cannot be changed via preferred node settings. Anonymous users viewing the local dashboard cannot control which nodes appear as "selected" or "unlocked" — this is a Cloud-only feature.

Reviewer: ```suggestion
Preferred node selection only affects Netdata Cloud dashboards. On the local agent dashboard (accessed directly at `http://<agent-ip>:19999`), the nodes shown in multi-node views are determined by the agent's streaming configuration and cannot be changed via preferred node settings.
```

### Increase Node Limits

> The 5-node limit on multi-node dashboards applies to both **Anonymous** and **Community** access levels. To work around this limit:

Reviewer: ```suggestion
The 5-node limit on multi-node dashboards applies to both **Anonymous** and **Community** users.
```

1. **Upgrade to Business** for unlimited nodes in multi-node views
2. **Select preferred nodes** in Space settings to choose which 5 nodes appear in multi-node views

:::note

Preferred nodes is a Netdata Cloud feature that requires signing in. It is **not available** to anonymous users accessing the local Agent dashboard directly.

:::

#### Workarounds for Anonymous Users

If you access the dashboard without signing in, you can:

- **Sign in to Netdata Cloud** (free Community plan) and use preferred nodes in Space settings to control which 5 nodes appear
- **Access individual Agent dashboards directly** by navigating to each Agent's URL (e.g. `http://<agent-ip>:19999`) — single-node dashboards have no node limit

> - **Upgrade to Business** for unlimited nodes across all dashboards

Reviewer: ```suggestion
```
